okhttp-tests/src/test/java/com/squareup/okhttp/internal/DiskLruCacheTest.java

```diff
@@ -1225,15 +1207,23 @@ private void assertAbsent(String key) throws Exception {
 
   private void assertValue(String key, String value0, String value1) throws Exception {
     DiskLruCache.Snapshot snapshot = cache.get(key);
-    assertEquals(value0, snapshot.getString(0));
-    assertEquals(value0.length(), snapshot.getLength(0));
-    assertEquals(value1, snapshot.getString(1));
-    assertEquals(value1.length(), snapshot.getLength(1));
+    assertSnapshotValue(snapshot, value0, 0);
+    assertSnapshotValue(snapshot, value1, 1);
     assertTrue(getCleanFile(key, 0).exists());
     assertTrue(getCleanFile(key, 1).exists());
     snapshot.close();
   }
 
+  private void assertSnapshotValue(DiskLruCache.Snapshot snapshot, String value, int index)
```

Comment:
nit: typically it's (index, value), not (value, index). As in List.set() among others.